converted-ostack-docs/ceilometer-xml/admin-telemetry-best-practices.xml

```diff
@@ -73,7 +73,7 @@
                         <literal>WSGIDaemon</literal>.</para>
         <note>
           <para>For more information on how to configure <literal>mod_wsgi</literal>, see the
-                            <link xlink:href="https://docs.openstack.org/developer/ceilometer/install/mod_wsgi.html">Telemetry Install Documentation</link>.</para>
+                            <link xlink:href="https://docs.openstack.org/ceilometer/pike/">Telemetry Install Documentation</link>.</para>
```

Comment:
This one is on automatic redirect. I cannot find the old file in the /pike/ directory to save my life. I'm guessing it doesn't exist at all. Up to you @dmpop if you think this should be removed completely?